resources/files/windows/PuppetBolt/PuppetBoltFunctions.ps1

```diff
@@ -0,0 +1,1639 @@
+Add-Type -TypeDefinition @"
+public enum BoltOutputLevel
+{
+   Verbose,
+   Debug,
+   Trace
+}
+"@
+
+Add-Type -TypeDefinition @"
+public enum BoltRerunTypes
+{
+   all,
+   failure,
+   success
+}
+"@
+
+function Convert-BoltParams {
```

Comment:
No need to concatenate the name here.  Use the full `Convert-BoltParameters`

Reply:
May be best to use the `ConvertTo-***` name if you want to be verbose about what you're converting to.